components/raftstore/src/store/fsm/peer.rs

```diff
@@ -2487,7 +2488,7 @@ where
         }
 
         if msg.has_extra_msg() {
-            self.on_extra_message(msg);
+            self.on_extra_message(&mut msg);
```

Comment:
Why `&mut`?

Reply:
Sorry, forgot to revert `on_extra_message`'s signature.